src/sentry/receivers/transactions.py

```diff
@@ -0,0 +1,4 @@
+from sentry.ingest.transaction_clusterer.datasource.redis import record_transaction_name
+from sentry.signals import transaction_processed
+
+transaction_processed.connect(record_transaction_name, weak=False)
```

Comment:
@getsentry/sns I tagged you on this PR mainly to verify if this is the correct way to add business logic to post processing.